environment.yml

```diff
@@ -42,5 +45,6 @@ dependencies:
   # For CLI
   - rich-click
   - click
+  - pip  # else Docker raises a warning in CI builds
   - pip:
-      - doit==0.35.0
+      - doit>=0.35.0
```

Comment:
Now we can use the version on Conda-forge no?

Reply:
Yes we can, I'll update.